content/enterprise_influxdb/v1.9/tools/influxd-ctl.md

```diff
@@ -937,19 +937,31 @@ Show the contents of the backup.
 
 ###### [ `-newdb <newdb_name>` ]
 
-Name of the new database to restore to (must specify with `-db`).
+Name of the new database to restore to.
+(Requires the `-db` argument.)
 
 ###### [ `-newrf <newrf_integer>` ]
 
 Integer of the new [replication factor](/enterprise_influxdb/v1.9/concepts/glossary/#replication-factor) to restore to (this is capped to the number of data nodes in the cluster).
 
 ###### [ `-newrp <newrp_name>` ]
 
-Name of the new [retention policy](/enterprise_influxdb/v1.9/concepts/glossary/#retention-policy-rp) to restore to (must specify with `-rp`).
+Name of the new [retention policy](/enterprise_influxdb/v1.9/concepts/glossary/#retention-policy-rp) to restore to.
+(Requires the `-rp` argument)
+
+###### [ `-newduration <new_duration>` ]
+The shard duration when restoring.
+(Requires the `-rp` argument.)
+
+###### [ `-newshard <new_shard_name` ]
+
+Name of a new destination shard ID, if different from the old one.
```

Comment:
Maybe 'if the target shard differs from the shard id in the backup'.

Reply:
Cool, thanks. 👍